extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts

```diff
@@ -12,9 +12,11 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 	private _optionsLoadingBuilder: azdata.LoadingComponentBuilder;
 	private _onValueChangedEmitter: vscode.EventEmitter<void> = new vscode.EventEmitter();
 	private _currentRadioOption!: azdata.RadioButtonComponent;
+	private _enabled: boolean;
 	constructor(private _view: azdata.ModelView, private _onNewDisposableCreated: (disposable: vscode.Disposable) => void, private _fieldInfo: FieldInfo) {
 		this._optionsDivContainer = this._view!.modelBuilder.divContainer().withProps({ clickable: false }).component();
 		this._optionsLoadingBuilder = this._view!.modelBuilder.loadingComponent().withItem(this._optionsDivContainer);
+		this._enabled = instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? false : <boolean>this._fieldInfo.enabled;
```

Comment:
Do you need the cast? The type is already boolean so that shouldn't be necessary (`_enabled` should be type `boolean | undefined` to match the type in the component properties - we want to let the component determine the default enabled state)